tools/autograph-monitor/selftest-entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -o pipefail
+
+docker compose run monitor
```

Comment:
I don't follow how this works. This seems to be running docker compose from within a dockerfile?

... are the CI integration tests actually running this code?

Reply:
Hah! Copy paste issue. I meant to grab `/go/bin/autograph-monitor`. Although I suppose that could work with a `dind` image.

Reply:
Why didn't the integration tests fail on this PR? I don't think this code is being run in CI and we would like it to, no?

Reply:
It used to override the command to do `/go/bin/autograph-monitor`. I've adjusted the docker-compose and workflow to use the default command, which is now `selftest-entrypoint.sh`.

Reply:
Ah, because the docker-compose.yaml used to have `command: ["/go/bin/autograph-monitor"]` and it was bypassing that script. But I wanted that script to make things a little more explicit on an error exit. I've removed that and now we're using the default command from the dockerfile (which runs this script).

Reply:
I don't follow. I don't see any changes in the "Fixing entrypoint bug" commit that looks like that and that was the one that was fixing the typo.